examples/mcmc.jl

```diff
@@ -137,26 +127,27 @@ function data_generating_procedure(t::Int64)
         _choices = Gen.get_submap(choices, addr)
         Gen.set_submap!(obs, addr, _choices)
     end
-
-    #=  plt = lineplot(1:t, get_zs(trace),
-                   title="Height of ball", name="observation with res $(trace[:prior => 1 => :restitution])",
-                   xlabel="t", ylabel="z", canvas=DotCanvas,
-                   border=:ascii) =#
-    plt = plot(1:t, get_zs(trace),title="Height of ball", xlabel="t", ylabel="z",label="Observation")
     
-    return (gargs, obs, plt)
+    return (gargs, obs)
 
 end
 
 function main()
 
     t = 60 # 1 second of observations
-    (gargs, obs, plt) = data_generating_procedure(t)
+    (gargs, obs) = data_generating_procedure(t)
+
+    zs = [get_submap(obs, :kernel => i => :observe => 1)[:position][3] for i in 1:t]
```

Comment:
`get_submap` is better used to collect sets of data. here you are calling it to get the positions at each time step. you could simply just use 

```julia
map(i -> obs[:kernel => i => :observe => 1], 1:t)
```